framework/contrib/hit/main.cc

```diff
@@ -267,6 +279,8 @@ format(int argc, char ** argv)
     return 1;
   }
 
+  int maxlen = std::stoi(flags.val("maxlen"));
```

Comment:
What about 5elephant? How many times are we going to write this same bug?

Reply:
This is sort of just a play utility I've used for dinking around testing/experimenting with hit stuff - not any sort of official binary meant to be used by others.